docs/setup/settings.asciidoc

```diff
@@ -592,6 +592,9 @@ The `server.xsrf.whitelist` setting requires the following format:
 [cols="2*<"]
 |===
 
+| `status.xsrf.disableProtection:`
```

Comment:
How about adding an anchor on line 591 (above the table)  so the link in the other files can go directly to this setting.  So add this:

[[settings-xsrf]]

And then change your links to this:

<<settings-xsrf, `server.xsrf.disableProtection`>>

Reply:
Well you already approved this--
Looking at the rest of the settings file, it appeared that you could add an anchor tag within the table cell. So I added one for each of the XSRF settings :) great suggestion, thanks! TIL